tests/services/test_schemas/test_affiliations.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: File naming wise I would tink it should be ``tests/services/schemas/test_affilations.py`` Remember that ``test_affilations.py`` has to be globally unique in all test folders.

Reply:
True, I will rename the folder in a proceeding PR if you agree.